application/Views/errors/html/error_exception.php

```diff
@@ -86,12 +86,15 @@
 									<div class="args" id="<?= $args_id ?>">
 										<table cellspacing="0">
 
-										<?php foreach ($row['args'] as $key => $value) : ?>
-											<?php
-												$mirror = isset($row['class']) ? new \ReflectionMethod($row['class'], $row['function']) :
-													new \ReflectionFunction($row['function']);
-												$params = $mirror->getParameters();
-											?>
+										<?php
+										$params = null;
+										// Reflection by name is not available for closure function
+										if( substr( $row['function'], -1 ) !== '}' )
```

Comment:
I don't know if it's a concern or not, but should we trim that before relying on the last character?